kotlin/display-grid/src/main/res/layout/popup_menu.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="16dp"
+        android:text="@string/grid_type"
+        android:textSize="20sp"
+        android:id="@+id/grid_type"
+        android:textColor="#000000"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginTop="16dp"
+        android:layout_marginLeft="16dp" android:layout_marginStart="16dp"/>
+
+    <Spinner
+        android:id="@+id/layer_spinner"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentBottom="true"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintBottom_toBottomOf="@+id/grid_type"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/grid_type"
+        app:layout_constraintTop_toTopOf="@+id/grid_type"></Spinner>
+
+    <CheckBox
+        android:id="@+id/labels_checkBox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentBottom="true"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:button="@null"
+        android:drawableRight="?android:attr/listChoiceIndicatorMultiple"
+        android:text="@string/labels_visible"
+        android:textSize="20sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/label_position_text" />
+
+    <TextView android:layout_width="wrap_content" android:layout_height="wrap_content"
+        android:text="@string/grid_color"
+        android:layout_marginStart="16dp"
+        android:textSize="20sp"
+        android:textColor="#000000"
+        android:id="@+id/textView"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginTop="16dp"
```

Comment:
was it this 16dp that @TADraeseke was referring to that we can avoid in samples going forward (something about screen size limit in earlier samples?)

Reply:
I think he was referring to older samples with margins around the mapView? Unsure. Actually, I think this spinner could probably use a 16dp margin around the bottom?